asdf/tests/test_asdf.py

```diff
@@ -100,3 +100,23 @@ class FooExtension:
         with pytest.raises(TypeError):
             with open_asdf(path, extensions=arg) as af:
                 pass
+
+
+def test_serialization_context():
+    context = SerializationContext("1.4.0")
+    assert context.version == "1.4.0"
+    assert context.extensions_used == set()
+
+    extension = get_config().extensions[0]
```

Comment:
What extension is used here?

Reply:
It depends on the entry points load order and what the user has installed.  For the purposes of this test it just needs to be any valid extension.  There will always be at least one due to the built-in extension.